src/components/Offerings/index.web.js

```diff
@@ -13,7 +13,7 @@ function Offerings() {
 					through active advising and coherent customer focused strategies.
 				</h2>
 			</div>
-			<div className={styles.offerings}>
+		{/*<div className={styles.offerings}>
```

Comment:
Do you want to comment all offerings or just one? You opened the comment before the "offerings" div, but closed it after a single "offering" div.

Use `npm start` to see what actually happens here.